src/Test.UnitTests.Sarif/Visitors/PerRunPerRuleSplittingVisitorTests.cs

```diff
@@ -0,0 +1,176 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using Microsoft.CodeAnalysis.Test.Utilities.Sarif;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.Visitors
+{
+    public class PerRunPerRuleSplittingVisitorTests
+    {
+        [Fact]
+        public void PerRunPerRuleSplittingVisitor_EmptyLog()
+        {
+            var visitor = new PerRunPerRuleSplittingVisitor();
+            visitor.VisitRun(new Run());
+            visitor.SplitSarifLogs.Count.Should().Be(0);
+        }
+
+        [Fact]
+        public void PerRunPerRuleSplittingVisitor_Simple()
```

Comment:
>PerRunPerRuleSplittingVisitor_Simple [](start = 20, length = 36)

these tests could be named more aptly - i.e. i cant tell quickly the difference between "simple" and "default".